source/core/parameters_multiphysics.cc

```diff
@@ -116,6 +122,18 @@ Parameters::VOF::parse_parameters(ParameterHandler &prm)
     peeling_wetting.parse_parameters(prm);
     surface_tension_force.parse_parameters(prm);
 
+    // DissipationIndicator
+    const std::string op = prm.get("viscous dissipative fluid");
```

Comment:
I'd use the full name of the parameter here, but not a big deal.

Reply:
I prefer to leave it with an "op" which is just used for the transition between the input string and the real parameter. It's the same machanism as for the "verbosity" parameter!